change/@fluentui-react-image-8016107b-e542-41f6-8df7-dd412c3bf263.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
This should `patch` or `prelease` as there are changes in component's code, we _want_ to bump it.

Reply:
Changed it to prelease, thanks